chainer/functions/activation/lstm.py

```diff
@@ -24,7 +25,10 @@ def _grad_tanh(x):
 
 
 _preamble = '''
-template <typename T> __device__ T sigmoid(T x) { return 1 / (1 + exp(-x)); }
+template <typename T> __device__ T sigmoid(T x) {
+    T half = 0.5;
```

Comment:
Is it okay to declare without `const`? In other words, does `nvcc` treat `half` as a constant?